src/components/Prescription/Prescription.css

```diff
@@ -34,7 +34,7 @@
 .top-container {
     display: flex;
     flex-direction: column;
-    justify-content: space-between;
+    justify-content: flex-start;
```

Comment:
I changed this because the link was showing up below the border when the show name was long, and it was unclickable. Now it appears directly under the show name